src/fparser/two/Fortran2003.py

```diff
@@ -5961,6 +6011,7 @@ def tofortran(self, tab='', isfix=None):
             lblock.append(item.tofortran(tab=tab+extra_tab, isfix=isfix))
         if len(self.content) > 1:
             lblock.append(end.tofortran(tab=tab, isfix=isfix))
+        py2_encode_list_items(lblock)
```

Comment:
Add a test for this as was done for r438 and r808?

Reply:
Done.